src/events.cr

```diff
@@ -138,7 +138,8 @@ module Office365::Events
     calendar_id : String? = nil,
     period_start : Time = Time.local.at_beginning_of_day,
     period_end : Time? = nil,
-    ical_uid : String? = nil
+    ical_uid : String? = nil,
+    top : UInt16? = 10000
```

Comment:
I think make this an `Int`
otherwise when using the lib you'll have to do:
`calendar_view_path("me@whatever", top: 200_u16)`
instead of just
`calendar_view_path("me@whatever", top: 200)`
and we really only care if it's an number